development/compiling/optimizing_for_size.rst

```diff
@@ -3,49 +3,195 @@
 Optimizing a build for size
 ===========================
 
-.. highlight:: shell
+.. highlight:: none
 
 Rationale
 ---------
 
 Sometimes, it is desired to optimize a build for size rather than speed.
 This means not compiling unused functions from the engine, as well as using
 specific compiler flags to aid on decreasing build size.
-Common situations include creating builds for mobile and Web platforms.
+Common situations include creating builds for mobile and web platforms.
 
 This tutorial aims to give an overview on different methods to create
 a smaller binary. Before continuing, it is recommended to read the previous tutorials
 on compiling Godot for each platform.
 
-.. seealso::
-
-    You can use the online
-    `Godot build options generator <https://godot-build-options-generator.github.io/>`__
-    to generate a ``custom.py`` file containing SCons options.
-    You can then save this file and place it at the root of your Godot source directory.
-
```

Comment:
It could be worded differently, but I think it's good to link to this resource early on, and not after users have already copied tons of options without understanding them from this page, which often end up outdated as this page isn't updated as often as the upstream code.